packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -374,209 +377,237 @@ export async function runApiExtraction({
       './dist-types',
       packageDir,
     );
+    const names = group.map(ep => ep.name);
 
-    const suffix = name === 'index' ? '' : `-${name}`;
-    const reportFileName = `api-report${suffix}.md`;
-    const reportPath = resolvePath(projectFolder, reportFileName);
-
-    const warningCountBefore = await countApiReportWarnings(reportPath);
+    const staleReportFiles = fs.readdirSync(projectFolder).filter(filename => {
+      const match =
+        filename.match(/^(.+)-api-report\.md$/) ||
+        filename.match(/^api-report-(.+)\.md$/);
+      return match && !names.includes(match[1]);
```

Comment:
This doesn't check whether `api-report.md` should exist. I'm thinking it's better to filter these based on the actual `reportFileName` below? Could make it a set here, then `staleReportFiles.delete(reportFileName)` in the loop, and check that it's empty afterwards.

Reply:
Aight, that does make things a bit neater too, fixed